src/main/java/net/snowflake/ingest/streaming/example/SnowflakeStreamingIngestExample2.java

```diff
@@ -0,0 +1,282 @@
+/*
+ * Copyright (c) 2021 Snowflake Computing Inc. All rights reserved.
+ */
+
+package net.snowflake.ingest.streaming.example;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.Map;
+import java.util.Properties;
+import java.util.concurrent.TimeUnit;
+import net.snowflake.ingest.streaming.InsertValidationResponse;
+import net.snowflake.ingest.streaming.OpenChannelRequest;
+import net.snowflake.ingest.streaming.SnowflakeStreamingIngestChannel;
+import net.snowflake.ingest.streaming.SnowflakeStreamingIngestClient;
+import net.snowflake.ingest.streaming.SnowflakeStreamingIngestClientFactory;
+import org.apache.hadoop.util.StopWatch;
+
+/**
+ * Example on how to use the Streaming Ingest client APIs.
+ *
+ * <p>Please read the README.md file for detailed steps
+ */
+public class SnowflakeStreamingIngestExample2 {
```

Comment:
Are you planning on checking this in?

Reply:
nope, this is for the private jar, as well as for others to take it, the base is 1.1.1, not master